README.md

```diff
@@ -1 +1,38 @@
-# Portfolio-project-
\ No newline at end of file
+## Portfolio project
+
+## Description the project.
+
+```
+In this project i learnt how to use and to design a website with figmas, how to use them and even created an account of figmq site, I enjoyed too again the use of linters and using git and github in a professional way
+
+```
+
+## Built With
+
+- html
+- css
+
+Authors
+👤 Author1
+
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+👤 Author2
+
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
```

Comment:
- [ ] This part of the readme document is not formated, please put links in Author section and indicate titles for each section eg: 
`## Authors`
`## Acknowledgments`
etc

Reply:
- [ ] As the previous reviewer told you, make sure that you put links in the author section. you can do that like this:
```
GitHub: [@AnicetFantomas ](https://github.com/anicetFantomas)
LinkedIn: [@AnicetFantomas ](https://www.linkedin.com/your-linkedin-adress/)
```